chucks-pick3-web/angular-app/src/app/models/customer-info.ts

```diff
@@ -0,0 +1,13 @@
+
+
+export interface CustomerInfo {
+
+  firstName: string;
+  lastName: string;
+  email: string;
+  phone: string;
+  city: string;
+  state: string;
+  id: string;
```

Comment:
Please use Integer or int instead of string.

Reply:
Changed id value to 'number'.